src/chainparams.cpp

```diff
@@ -179,9 +181,9 @@ class CTestNetParams : public CMainParams {
 
         checkpointData = (Checkpoints::CCheckpointData) {
             boost::assign::map_list_of
-            ( 546, uint256S("000000002a936ca763904c3c35fce2f3556c559c0214345d31b1bcebf76acb70")),
```

Comment:
Why remove the checkpoint? why not just add the genesis checkpoint as in https://github.com/jtimon/bitcoin/commit/dc1611925dd36f07d1cf3412237610b864a99bcc
Also, why not just use consensus.hashGenesisBlock in the genesis checkpoint instead of repeating the hardcoded value?